test/kube_testing/k8s.go

```diff
@@ -267,11 +267,10 @@ func NewK8s() (*K8s, error) {
 	return &client, nil
 }
 
-func (o *K8s) deletePods(pods ...*v1.Pod) error {
+func (o *K8s) deletePods(timeout int64, pods ...*v1.Pod) error {
```

Comment:
Is this needed for NSMD healing tests? If not strictly needed, I'd suggest to extract ot another PR.